PowerEditor/src/NppDarkMode.cpp

```diff
@@ -0,0 +1,608 @@
+#include "nppDarkMode.h"
+
+#include "DarkMode/DarkMode.h"
+#include "DarkMode/UAHMenuBar.h"
+
+#include <Uxtheme.h>
+#include <Vssym32.h>
+
+#include "Parameters.h"
+#include "resource.h"
+
+#include <Shlwapi.h>
+
+#pragma comment(lib, "uxtheme.lib")
```

Comment:
Why is it needed? Isn't it enough to add this static library to the linker options in project file?